paddle/fluid/framework/operator.cc

```diff
@@ -635,6 +635,15 @@ void OperatorWithKernel::RunImpl(const Scope& scope,
   VLOG(3) << "expected_kernel_key:" << expected_kernel_key;
 
   auto kernel_iter = kernels.find(expected_kernel_key);
+#ifdef PADDLE_WITH_MKLDNN
+  // workarkound for missing MKLDNN kernel when FLAGS_use_mkldnn env var is set
```

Comment:
typo: workaround 
Maybe can fix this typo at the same time: https://github.com/PaddlePaddle/Paddle/pull/11544#pullrequestreview-131558634

Reply:
done